docs/fsharp/language-reference/parameters-and-arguments.md

```diff
@@ -134,6 +134,16 @@ type C =
         printfn "%s" message
 ```
 
+You can also specify new objects as a default parameter value. For example, the `Foo` member could have an optional `CanceallationToken` as input instead:
```

Comment:
a new object as a default parameter value? The plural the singular don't go well together.